flutter/example/pubspec.yaml

```diff
@@ -11,15 +11,21 @@ environment:
 dependencies:
   flutter:
     sdk: flutter
+  sentry:
   sentry_flutter:
```

Comment:
instead of removing the relative path, I'd rather add for `sentry` too, I recall having issues when using `dependency_overrides` and using `pana`, so we keep the same as every other example too.

Reply:
That doesn't work because `sentry_flutter` pulls `sentry: ^x.x.x` - that results in a dependency conflict if it is specified as path dependency. It needs to be an override.

Also pana ist not really relevant for an app since it is not being published to `pub.dev` or am I missing something ?

Reply:
yeah, I didnt run the dependency conflict before so its fine.